app/controllers/Projects_Controller.rb

```diff
@@ -38,6 +38,15 @@ def show
     @project=Project.find(params[:id])
     @chatroom=Chatroom.find_by(slug: @project.id)
     @current_user=current_user
+    @owner=nil
+    current_user.owned_projects.each do |project|
+      if project.id == @project.id
+        @owner=true
+        break
```

Comment:
if the last project is not ==, it will set it to false. I just need the first project to ping correct for the chatroom button to show